demisto_sdk/utils/server_schema_change_slack_notifier.py

```diff
@@ -0,0 +1,137 @@
+"""Notify Slack channel SCHEMA_UPDATE_CHANNEL on schema struct file changes.
+
+Schema struct files are those who start with PREFIX and end with SUFFIX."""
+
+import argparse
+import os
+from pprint import pformat
+from typing import Any, Dict, List, Tuple
+
+from slack_sdk import WebClient
+
+SCHEMA_UPDATE_CHANNEL = "dmst-schema-change"
+GITHUB_BASE_URL = "https://github.com/demisto/server/commit/"
+PREFIX = "domain/"
+SUFFIX = "_struct.go"
+
+
+def options_handler():
+    parser = argparse.ArgumentParser(description='Slack notifier for changes in schema structs.')
+    parser.add_argument('-c', '--commit', help='The commit hash', required=True)
+    parser.add_argument('-s', '--slack_token', help='Token for slack messages', required=True)
+    options = parser.parse_args()
+    return options
+
+
+def extract_changes_from_commit(commit_hash: str) -> Tuple[str, List[str], dict, str]:
+    """Extract relevant changes from git history via commit.
+
+    Args:
+        commit_hash: current commit hash
+
+    Return: Tuple of
+        commit_github_url: Url of commit on github.
+        relevant_changed_files: List of relevant files that were changed.
+        change_diffs: Dictionary with changed_files keys and changesets as values.
+        commit_author: Author of commit.
+    """
+    commit_github_url = os.path.join(GITHUB_BASE_URL, commit_hash)
+    output_stream = os.popen("git log -1 --pretty=format:'%an'")
```

Comment:
might be easier to use gitPython here instead of all the streams - https://gitpython.readthedocs.io/en/stable/tutorial.html

Reply:
I tried but inside the server build I got an error 'git.exc.NoSuchPathError' but with a real path '/builds/xsoar/server'
This is probably somehow connected to the difference docker runs python code and os.popen.
I don't really want to get to the bottom of this since:
1. os.popen functions do work
2. The readability gitpython adds is minimal for my taste
3. Using gitpython adds another library installation in the build/ docker image which I think is unnecessary 
4. It would take up some time

You can checkout the code in the last commit to see if you think its worth the trouble: https://github.com/demisto/demisto-sdk/pull/1019/commits/86992acd911a3c98ecb2e870e3ecdbab3e3a5090

Reply:
I see - then please add a dev note as to what is being done here for readability and maintainability. - same for the other popen git call.